apps/extension/src/entrypoints/side.content/components/side-content/top-bar.tsx

```diff
@@ -208,3 +213,41 @@ function SourceLangSelect() {
     </Select>
   )
 }
+
+function FileExport() {
+  const explainDataList = useMutationState({
+    filters: {
+      mutationKey: ['explainArticle'],
+    },
+    select: mutation => mutation.state.data,
+  }) as ArticleExplanation['paragraphs']
```

Comment:
Can we remove the `as` assertion here? Is there another method to ensure the type? Using `as` can lead to bugs easily. so it's just should be treat as the last method